src/main/java/build/buf/protovalidate/Format.java

```diff
@@ -139,52 +140,25 @@ private static String bytesToHex(byte[] bytes, char[] digits) {
    * @param val the value to format.
    */
   private static void formatString(StringBuilder builder, Val val) {
-    if (val.type().typeEnum() == TypeEnum.String) {
-      builder.append(val.value());
-    } else if (val.type().typeEnum() == TypeEnum.Bytes) {
-      builder.append(new String((byte[]) val.value(), StandardCharsets.UTF_8));
-    } else if (val.type().typeEnum() == TypeEnum.Double) {
-      DecimalFormat format = new DecimalFormat();
-      builder.append(format.format(val.value()));
-    } else {
-      formatStringSafe(builder, val, false);
-    }
-  }
-
-  /**
-   * Formats a string value safely for other value types.
-   *
-   * @param builder the StringBuilder to append the formatted string to.
-   * @param val the value to format.
-   * @param listType indicates if the value type is a list.
-   */
-  private static void formatStringSafe(StringBuilder builder, Val val, boolean listType) {
     TypeEnum type = val.type().typeEnum();
     if (type == TypeEnum.Bool) {
       builder.append(val.booleanValue());
-    } else if (type == TypeEnum.Int || type == TypeEnum.Uint) {
-      formatDecimal(builder, val);
-    } else if (type == TypeEnum.Double) {
-      // When a double is nested in another type (e.g. a list) it will have a minimum of 6 decimal
-      // digits. This is to maintain consistency with the Go CEL runtime.
-      DecimalFormat format = new DecimalFormat();
-      format.setMaximumFractionDigits(Integer.MAX_VALUE);
-      format.setMinimumFractionDigits(6);
-      builder.append(format.format(val.value()));
     } else if (type == TypeEnum.String) {
-      builder.append("\"").append(val.value().toString()).append("\"");
+      builder.append(val.value());
     } else if (type == TypeEnum.Bytes) {
-      formatBytes(builder, val);
+      builder.append(new String((byte[]) val.value(), StandardCharsets.UTF_8));
+    } else if (type == TypeEnum.Int || type == TypeEnum.Uint || type == TypeEnum.Double) {
```

Comment:
Shouldn't we only need to format float/double as decimals? It feels like for int/uint we could just output the string value.